container.go

```diff
@@ -282,12 +275,12 @@ func getPersistentContainer(ctx context.Context, cli *client.Client, profile *Pr
 	f := filters.NewArgs()
 	f.Add("label", "com.viam.canon.type=persistent")
 	f.Add("label", "com.viam.canon.profile="+profile.name+"/"+profile.Arch)
-	containers, err := cli.ContainerList(ctx, types.ContainerListOptions{Filters: f})
+	containers, err := cli.ContainerList(ctx, container.ListOptions{All: true, Filters: f})
```

Comment:
and same question here. Neither [the docs on `ContainerList`](https://pkg.go.dev/github.com/docker/docker/client#Client.ContainerList) nor [the docs on `ListOptions`](https://pkg.go.dev/github.com/docker/docker@v25.0.3+incompatible/api/types/container#ListOptions) describe what these do.

Reply:
Annoying, isn't it, when docs are useless like that? :-)

I only "knew it" because it's how the `docker ps` command itself seems to work, and I verified that was basically calling this on the backend.

That said, I also learned (when originally writing canon) that the docker API docs can be more useful. https://docs.docker.com/engine/api/v1.44/#tag/Container/operation/ContainerList